packages/turf-boolean-point-on-line/index.js

```diff
@@ -0,0 +1,65 @@
+var getCoords = require('@turf/invariant').getCoords;
+
+/**
+ * Returns true if a point is on a line. Accepts a optional parameter to ignore the start and end vertices of the linestring.
+ *
+ * @name booleanPointOnLine
+ * @param {Geometry|Feature<Point>} point GeoJSON Feature or Geometry
+ * @param {Geometry|Feature<LineString>} linestring GeoJSON Feature or Geometry
+ * @param {Boolean} [ignoreEndVertices=false] whether to ignore the start and end vertices.
+ * @returns {Boolean} true/false
```

Comment:
I remember having this conversation with @tmcw about `Boolean` & `boolean`.

From what I can read on JSDocs, it's saying to use `boolean`
http://usejsdoc.org/tags-type.html

Also DocumentationJS linting flags `Boolean` as a warning (which we will be adding soon)

This change applies to many modules, so don't need to fix this right now.